docs/getting_started.md

```diff
@@ -0,0 +1,31 @@
+# 📚 Overview
+
+Dozer is a Real-Time SQL Engine that connects to any of your data sources, transforms and stores the data in an embedded cache.
```

Comment:
Dozer is not just a real time sql engine. Dozer is a Data Product backend that embeds a streaming SQL engine, a low-latency data cache and generates gRPC and REST APIs that can be easily integrated in frontend applications.